aptos-move/framework/src/natives/event.rs

```diff
@@ -83,15 +130,30 @@ pub fn make_all(
     timed_features: TimedFeatures,
     features: Arc<Features>,
 ) -> impl Iterator<Item = (String, NativeFunction)> {
-    let natives = [(
-        "write_to_event_store",
-        make_safe_native(
-            gas_params.write_to_event_store,
-            timed_features,
-            features,
-            make_native_write_to_event_store(calc_abstract_val_size),
+    let natives = [
+        (
+            "write_to_event_store",
+            make_safe_native(
+                gas_params.write_to_event_store,
+                timed_features.clone(),
+                features.clone(),
+                make_native_write_to_event_store(calc_abstract_val_size),
+            ),
+        ),
+        (
+            "emitted_internal",
+            make_safe_native(
+                (),
+                timed_features.clone(),
+                features.clone(),
+                native_emitted_internal,
+            ),
+        ),
+        (
+            "emitted_event_internal",
+            make_safe_native((), timed_features, features, native_emitted_event_internal),
         ),
```

Comment:
Weird. Why has the [check-vm-features](https://github.com/aptos-labs/aptos-core/actions/runs/5605042252/jobs/10253708618?pr=9181#logs) test succeed even though you are not feature gating these natives here...

Reply:
@davidiw once you fix the check, you'll need feature gate these natives here via `#[cfg(feature = "testing")]`.

Reply:
addressed